src/main/java/Deadline.java

```diff
@@ -0,0 +1,63 @@
+public class Deadline extends TaskList{
+    private static final String TASKTITLE ="D";
```

Comment:
Inconsistent spacing with regards to operator. Should be 1 spacing before and after operator.

Reply:
Noted!!!